### PR TITLE
net-mail/pflogsumm: use EAPI 7

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks

### CODE REVIEW

File: net-mail/pflogsumm/pflogsumm-1.1.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Pflogsumm is a log analyzer for Postfix logs"
+HOMEPAGE="https://jimsun.linxnet.com/postfix_contrib.html"
+SRC_URI="https://jimsun.linxnet.com/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
```
Comment (mgorny) [inline net-mail/pflogsumm/pflogsumm-1.1.5-r1.ebuild]: GPL-2+

**Comment by hydrapolic** [comment]
> Why don't you remove the mask while at it?

done

File: net-mail/pflogsumm/pflogsumm-1.1.5-r1.ebuild (hunk 1 above)
Comment (hydrapolic) [reply net-mail/pflogsumm/pflogsumm-1.1.5-r1.ebuild]: Changed, thanks

**Comment by hydrapolic** [comment]
Thanks @mgorny

**Pull request closed without merging**